### PR TITLE
Add zIndex to CZML

### PR DESCRIPTION
**Pull request by hpinkos**
Applicable to `corridor`, `ellipse`, `polygon` and `rectangle`.

CC @shunter

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@hpinkos, thanks for the pull request! Maintainers, we have a signed CLA from @hpinkos, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
Since we merged in @likangning93 CZML changes, I want to get this merged in for this release too.

@ggetz can you review?

File: Apps/Sandcastle/gallery/CZML ZIndex.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="CZML Circles and Ellipses">
```
Comment (likangning93) [inline Apps/Sandcastle/gallery/CZML ZIndex.html]: Update description?

**Comment by hpinkos** [comment]
@likangning93 ready.  I also made some tweaks to `CHANGES.md` since there was a lot of duplication for your two polyline on terrain bullet points, and overlap between your CZML PR and mine.

**Comment by ggetz** [comment]
Looks good to me! Thanks @hpinkos!

**Comment by shunter** [comment]
@hpinkos - is zIndex supposed to be an integer or a floating point value? Digging through the code it's not entirely clear.

**Pull request merged**